### PR TITLE
Update configure-oauth-authentication-between-exchange-and-exchange-o…

### PR DESCRIPTION
**Pull request by mukeshHCW**
…nline-organizations-exchange-2013-help.md

Change from verified domain to tenant's coexistence domain. This is required to ensure manual Oauth configuration is exactly similar to HCW Oauth configuration.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer2013/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help.md [hunk 1]
```diff
@@ -55,18 +55,21 @@ For this procedure, you have to specify a verified domain for your Exchange Onli
...
 > ```Powershell  
-> New-AuthServer -Name "WindowsAzureACS" -AuthMetadataUrl "https://login.microsoftonline.us/<your verified domain>/metadata/json/1"  
-> New-AuthServer -Name "evoSTS" -Type AzureAD -AuthMetadataUrl "https://login.microsoftonline.us/<your verified domain>/federationmetadata/2007-06/federationmetadata.xml"  
+> New-AuthServer -Name "WindowsAzureACS" -AuthMetadataUrl "https://login.microsoftonline.us/<your tenant coexistence domain>/metadata/json/1"  
+> New-AuthServer -Name "evoSTS" -Type AzureAD -AuthMetadataUrl "https://login.microsoftonline.us/<your tenant coexistence domain>/federationmetadata/2007-06/federationmetadata.xml"  
 > ```  
-
+   
+> [!NOTE]
+> Tenant coexistence domain is of the form contoso.mail.onmicrosoft.com
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/configure-oauth-authentication-between-exchange-and-exchange-online-organizations-exchange-2013-help.md:71]:
```suggestion
> The tenant coexistence domain is of the form contoso.mail.onmicrosoft.com.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by mukeshHCW** [comment]
@Kateyanne : The public facing page still shows the old version of information. When will that be updated?

**Comment by scanum** [comment]
Hi @mukeshHCW - Hope you are well. We have reviewed this issue and the PR was submitted on the wrong branch, you will need to submit a new PR on the public branch in order for us to be able to merge it correctly. Please let us know if there are any questions. Thanks

**Comment by mukeshHCW** [comment]
@scanum : Can you share the link of Public Branch where i can submit changes.

**Comment by Kateyanne** [comment]
Hi – this has been merged. Thank you, Kateyanne

**Pull request merged**